packages/errors/src/create-error.ts

```diff
@@ -0,0 +1,35 @@
+export interface DirectusError<Extensions = void> extends Error {
+	extensions: Extensions;
+	code: string;
+	status: number;
+}
+
+export interface DirectusErrorConstructor<Extensions = void> {
+	new (extensions: Extensions, options?: ErrorOptions): DirectusError<Extensions>;
+	readonly prototype: DirectusError<Extensions>;
+}
+
+export const createError = <Extensions = void>(
+	code: string,
+	message: string | ((extensions: Extensions) => string),
+	status = 500
+): DirectusErrorConstructor<Extensions> => {
+	return class extends Error implements DirectusError<Extensions> {
+		override name = 'DirectusError';
+		extensions: Extensions;
+		code = code.toUpperCase();
+		status = status;
+
+		constructor(extensions: Extensions, options?: ErrorOptions) {
```

Comment:
`ErrorOptions` seems to be not defined 🤔

Reply:
Type comes from default lib, so it should be available 👍 Maybe dependencies weren't installed on your end? 🤔